apps/web/src/components/user/profile.tsx

```diff
@@ -14,6 +14,9 @@ export const metadata: Metadata = {
 
 export async function Profile({ username: usernameFromQuery }: Props) {
   const [, username] = decodeURIComponent(usernameFromQuery).split('@');
+  
+  // This throws an error to reflect the behavior of findFirstOrThrow, ideally we will move this to notFound()
+  if(username === undefined) throw new Error("[ERROR]: Username not found, accessed profile page without @");
```

Comment:
figure out how we can get tailwind to work on the 404 page or not used the global layout if ya want!

Reply:
aight imma look into it!

Reply:
404 has been fixed by @Hacksore